pandas/tseries/resample.py

```diff
@@ -395,8 +397,23 @@ def _groupby_and_aggregate(self, how, grouper=None, *args, **kwargs):
 
         return self._wrap_result(result)
 
-    def _apply_loffset(self, result):
-        """if loffset if set, offset the result index"""
+    def _apply_loffset(self, result, arg=None):
+        """
+        if loffset is set, offset the result index
+
+        Parameters
+        ----------
+        result : Series or DataFrame
+            the result of resample
+        arg : string, list, or dict, optional
```

Comment:
this needs to be refactored to not do this. It is unlike anything else in pandas to pass an argument to determine state.